package.json

```diff
@@ -7,6 +7,7 @@
     "react": "^16.11.0",
     "react-dom": "^16.11.0",
     "react-redux": "^7.1.1",
+    "react-router-dom": "^5.1.2",
```

Comment:
do you need this?

Reply:
We were going to implement react-router-dom but are moving it to the navbar feature instead. Should we remove it from the package.json?

Reply:
yes

Reply:
Please address the changes

Reply:
Fixed